### DISCUSSION TITLE
Crear enlaces de descarga directa en Google Drive

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#4] [Announcements]
# Crear enlaces de descarga directa en Google Drive

Hoy te voy a enseñar a crear un enlace de descarga directo de un archivo en Google Drive.

https://www.zkreations.com/2018/01/como-crear-enlaces-descarga-directa-google-drive.html

**Comment by TakoMics** [comment]
Desde que leí esta entrada hace ya bastante tiempo, ese es el sistema que uso para enviar archivos directos en mi trabajo y a empleadores cuyo acceso al link fuente no quiero darles.

Hasta me ha funcionado para archivos propios. Es uno de los códigos que sí o sí siempre tengo en la nube por si requiero utilizarlo.

Gracias por renovar el tutorial.

**Reply by danieIabel** [reply]
Que tal amigo, es bueno leerte! con respecto a Google drive, al dia de hoy como podrás notar en el artículo a cambiado para bien, paso de **25mb** como tamaño límite (para descarga directa) a **100mb** y su ancho de banda es de 750gb diarios (se ocupa tanto para subida como descargas). Es una brutalidad de capacidad y gratis.

**Reply by TakoMics** [reply]
Sí y realmente cae genial ese update que le han hecho. Por lo menos para guardar archivos importantes y puntuales, cae excelente.

Recuerdo cuando lo empecé a usar. Muchos de mis contactos y clientes se sorprendían de que les pasara un enlace de Google Drive y al clickearlo descargara de una vez sin pasar por el Drive.

Gracias por la actualización de este material, Daniel. También es bueno leerte y tenerte activo por acá.

**Reply by danieIabel** [reply]
Ahora si tengo contenido tanto anterior (que estoy actualizando) como nuevo para rato amigo, mas ahora que le agarre un amor al codigo abierto, por lo tanto casi todo lo nuevo es de libre uso.

**Comment by carlos60benavides** [comment]
Hola Daniel.

Muchas gracias por el excelente artículo.

Tengo una pregunta... ¿Una vez que tenga el enlace, cómo hago para poner el audio y el video exactamente como está en este artículo?

De antemano muchas gracias.

**Reply by danieIabel** [reply]
Que tal amigo, claro no hay problema, yo estoy usando las etiquetas de audio y video de HTML integras, es decir, sin ninguna libreria ni codigo. Para audio seria:

```html
<audio controls>
  <source src="tu-enlace-de-drive" type="audio/mp3">
</audio>
```

En el caso de los videos:

```html
<video controls poster="tu-imagen-de-vista-previa.jpg">
  <source src="tu-enlace-de-drive" type="video/mp4">
</video>
```

**Reply by carlos60benavides** [reply]
Genial, muchísimas gracias Daniel, esto es muy útil. 👍

**Reply by danieIabel** [reply]
Gracias a ti por pasar y comentar amigo.

**Comment by aharen71** [comment]
Hay alguna forma de quitar el limitador de one drive?

**Reply by danieIabel** [reply]
Que tal amigo, quizas se refiere a Google Drive que el servicio usado en este artículo. Solo puede obtener más espacio adquiriendo alguno de los planes de google workspace, pero perdería el propósito de este artículo. Si está pensando en realizar algo que demanda más recursos, en ese caso le recomiendo adquirir los servicios de un hosting.

**Reply by aharen71** [reply]
Cómo que hosting me recomienda es para poder subir videos pues mis videos pesan más de 100 MB y pues con Google drive no se puede😢

**Reply by danieIabel** [reply]
Que tipo de página tiene amigo, es decir, ¿es una pagina de videos que requiere un gran número de estos, o solo necesita subir unos cuantos?

**Reply by aharen71** [reply]
Son videos acerca de como descargar desde aplicaciones y juegos y pues la mayoría de veces pesan más de 100 MB pues uso one drive pero veo que one drive tiene límites ya que después los videos ya no se reproducen por qué dice que el límite se a alcanzado y ningún vídeo se reproduce después, y pues en one drive compré un poco las de almacenamiento y con ese límite pues no siento que valga la pena pagar ahí

**Reply by danieIabel** [reply]
Por supuesto que no amigo, este tipo de trucos son útiles para una pequeña cantidad de archivos, muy ocasionales. En este caso yo le recomendaría usar youtube, no tiene ningún límite de reproducción y si le preocupan las cookies, youtube ahora también brinda una opción sin cookies para insertar los videos. También puede ocultarlos de su canal marcando la opción "no listado".

Otra opción que tiene es subir el video a Blogger mediante la opción "Video" en el editor de Blogger, perderá calidad, pero tampoco tiene límites de visualización.

**Reply by aharen71** [reply]
Tenía ahí pero YouTube los eliminaba bueno los elimina z hay algúna manera de hacer los mismo con MediaFire así como con Google drive osea crear enlaces de descarga directa ahí?

**Reply by danieIabel** [reply]
En teoría con mediafire no se puede ya que genera un enlace de descarga directa distinto cada vez que se descarga un archivo. Si prefiere, puede contratar un hosting, no le puedo recomendar alguno ya que no creo que los que yo uso sean los adecuado para los videos de su proyecto, solo le puedo recomendar que elija uno que permita sus archivos y que a su vez disponga de banda ancha ilimitada.

**Comment by danieIabel** [comment]
Al parecer debido a los cambios con el tema de las cookies, los enlaces directos de Google Drive han dejado de funcionar. Estaré intentando actualizar el artículo con una alternativa. Mientras tanto, dejo una lista de otras opciones que puedes probar:

[Crear y usar enlaces directos de Dropbox](https://www.zkreations.com/2018/12/como-crear-enlaces-descarga-directa-dropbox.html)
[Crear enlaces de descarga directa en OneDrive](https://www.zkreations.com/2024/02/crear-enlace-descarga-directa-onedrive.html)
